sci-mathematics/rstudio/files/rstudio-1.4.1106-soci-cmake-find_library.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/src/cpp/CMakeLists.txt b/src/cpp/CMakeLists.txt
+index df54994..eda8c8f 100644
+--- a/src/cpp/CMakeLists.txt
++++ b/src/cpp/CMakeLists.txt
+@@ -409,9 +409,9 @@ if(UNIX)
+    if(NOT APPLE AND RSTUDIO_USE_SYSTEM_SOCI)
+       set(SOCI_LIBRARY_DIR "/usr/lib")
```

Comment:
I wonder if this is going to cause issues for us, not respecting libdir.

Reply:
Can you elaborate on what you mean by this?

Reply:
I think I clarified on IRC in the end

Reply:
Yes you did, I will fix this.

Reply:
@thesamesam  I looked in to this: per the [cmake documentation][1], the `PATHS` option simply adds extra places to search. The main point of my patch is to remove `NO_DEFAULTS` - the cmake `find_library` is smart enough to figure out where the libraries are (usually).

TL;DR this should not be an issue, as it is only used by cmake to find the library.

[1]: https://cmake.org/cmake/help/latest/command/find_library.html